common/src/main/java/com/microsoft/identity/common/adal/internal/cache/StorageHelper.java

```diff
@@ -628,36 +707,38 @@ private SecretKey generateSecretKey() throws NoSuchAlgorithmException {
     @TargetApi(Build.VERSION_CODES.JELLY_BEAN_MR2)
     private synchronized SecretKey getUnwrappedSecretKey()
             throws GeneralSecurityException, IOException {
-        Log.v(TAG, "Reading SecretKey");
+        final String methodName = ":getUnwrappedSecretKey";
+        Logger.verbose(TAG + methodName, "Reading SecretKey");
 
         final SecretKey unwrappedSecretKey;
         try {
             final byte[] wrappedSecretKey = readKeyData();
             unwrappedSecretKey = unwrap(wrappedSecretKey);
-            Log.v(TAG, "Finished reading SecretKey");
+            Logger.verbose(TAG + methodName, "Finished reading SecretKey");
         } catch (final GeneralSecurityException | IOException ex) {
             // Reset KeyPair info so that new request will generate correct KeyPairs.
             // All tokens with previous SecretKey are not possible to decrypt.
-            //Log.e(TAG, "Unwrap failed for AndroidKeyStore", "",ADALError.ANDROIDKEYSTORE_FAILED, ex);
-            Log.e(TAG, ErrorStrings.ANDROIDKEYSTORE_FAILED);
+            Logger.error(TAG + methodName, ErrorStrings.ANDROIDKEYSTORE_FAILED, ex);
             mKeyPair = null;
             deleteKeyFile();
             resetKeyPairFromAndroidKeyStore();
-            Log.v(TAG, "Removed previous key pair info.");
+            Logger.verbose(TAG + methodName, "Removed previous key pair info.");
             throw ex;
         }
 
         return unwrappedSecretKey;
     }
 
     private void deleteKeyFile() {
+        final String methodName = ":deleteKeyFile";
+
         // Store secret key in a file after wrapping
         final File keyFile = new File(mContext.getDir(mContext.getPackageName(),
                 Context.MODE_PRIVATE), ADALKS);
         if (keyFile.exists()) {
-            Log.v(TAG, "Delete KeyFile");
+            Logger.verbose(TAG + methodName, "Delete KeyFile");
             if (!keyFile.delete()) {
-                Log.v(TAG, "Delete KeyFile failed");
+                Logger.verbose(TAG + methodName, "Delete KeyFile failed");
```

Comment:
>failed [](start = 65, length = 6)

Does it make sense to log the reason why it failed?

Reply:
The delete() method won't tell us why it fails.